pandas/core/arrays/datetimes.py

```diff
@@ -577,15 +577,7 @@ def time(self):
         """
         Returns numpy array of datetime.time. The time part of the Timestamps.
```

Comment:
Could you expand this docstring to also mention that the `datetime.time` objects are in local time with associated timezone information.

Reply:
Does that look good?

Reply:
Small nit: could you replace `, and contains` with `with`. Otherwise LGTM

Reply:
No problem